vllm/distributed/device_communicators/custom_all_reduce.py

```diff
@@ -28,6 +28,10 @@ def _can_p2p(rank: int, world_size: int) -> bool:
     for i in range(world_size):
         if i == rank:
             continue
+        if envs.VLLM_SKIP_P2P_CHECK:
+            logger.info(
+                "Skipping P2P check and trusting the driver's P2P report.")
+            return torch.cuda.can_device_access_peer(rank, i)
```

Comment:
how long will this take? `torch.cuda.can_device_access_peer(rank, i)`

Reply:
this is just one api call, probably less than `ms` .